LoopKitUI/Views/ExpandableDatePicker.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ExpandableDatePicker.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 8/12/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+
+public struct ExpandableDatePicker: View {
+    @State var date: Date = Date()
+    let pickerRange: ClosedRange<Date>
+    let text: String
+    var onUpdate: (Date) -> Void
+    
+    public init (
+        with date: Date,
+        pickerRange: ClosedRange<Date>? = nil,
+        text: String = "",
+        onUpdate: @escaping (Date) -> Void
+    ) {
+        let today = Date()
+        self.pickerRange = pickerRange ?? today.addingTimeInterval(-.hours(6))...today.addingTimeInterval(.hours(6))
+        self.text = text
+        self.onUpdate = onUpdate
+        self.date = date
+    }
+    
+    public var body: some View {
+        ZStack(alignment: .topLeading) {
+            // ANNA TOOD: fix buggy animations
```

Comment:
The animations for the date picker appear a bit weirdly (though not for the other picker for some reason...). I can't find a way to disable the animation (setting `.animation(nil` doesn't do anything), but if you have ideas I'd love to find a fix